sandbox/src/timebar/timebar.js

```diff
@@ -172,9 +202,55 @@ class TimebarContainer extends Component {
                 />
               )
             }
-            return <TimebarActivity key="activity" {...props} activity={activity} />
+            if (currentChart === 'track') {
+              return <>
+                <TimebarActivity
+                  {...props}
+                  key="trackActivity"
+                  color="#fe81eb"
+                  opacity={.4}
+                  curve="curveBasis"
+                  activity={activityMockForSubchart}
+                />
+                <TimebarTracks
+                  key="tracks"
+                  tracks={[trackMockForSubchart[0]]}
+                  outerScale={outerScale}
+                  graphHeight={graphHeight}
+                />
+                <TimebarHighlighter
+                  graphHeight={props.graphHeight}
+                  outerScale={props.outerScale}
+                  tooltipContainer={props.tooltipContainer}
+                  hoverStart={hoverStart}
+                  hoverEnd={hoverEnd}
+                  activity={activityMockForSubchart}
+                  unit={(currentSubChart === 'courses' ? 'degrees' : 'knots')}
```

Comment:
@nerik shouldn't the units be taken from the header? This makes the units not to be that flexible.

Reply:
@enriquetuya You are right, will discuss in map-client.